plugin/adsk/scripts/USDMenuProc.mel

```diff
@@ -105,6 +105,9 @@ global proc USDMenuProc(string $parent, string $obj)
 {
     $obj = expandToSelection($obj);
 
+    popupMenu -e -dai $parent;
+    setParent -menu $parent;
+    
```

Comment:
The previous code also had `setParent -menu ..;`

Note: this doesn't actually create the menu - it is created by Maya. What this is doing is clearing out the menu and then setting it for subsequent `menuItem` calls.

Reply:
A! Well, the bug was in Maya, buildNonMayaContextMenu calls MEL command that fail because there are no menu active. So something in those two lines does the trick. I wrote those because they were in here before my previous change to use the UFE context menu, so I thought they were needed.

Reply:
If you got the right line, we can commit it with "suggested code changes" in this review.

Reply:
I'm actually confused why the previous code also had `setParent -menu ..` - the line above set the menu parent and then this one would make it move up one level.

Can you do a test and print out what the original parent is and what the calls to the two different `setParent` would be?

Reply:
The problem being fixed here is that as reported sometimes the parent is not a menu and does not have a popup menu. The version without the popuMenu call would only work by a happy accident most of the time. I'll check what were the parent when it work and did not work.

Reply:
he parent being passed in is for example "MainPane|viewPanes|modelPanel4|modelPanel4|modelPanel4|modelPanel4ObjectPop". What is needed is to set it as the parent to be used by the menu being created.

Reply:
Thanks for checking. I wonder if we should leave in the `popupMenu -e -dai $parent;` which would clear the menu of all items (in case it has any) as we are going to populate it with MayaUsd menu items.

Reply:
OK, I'll put it back then.